packages/flutter/test/rendering/viewport_test.dart

```diff
@@ -1843,6 +1843,189 @@ void main() {
     });
   });
 
+  Widget _buildShrinkWrap({
+    ScrollController? controller,
+    Axis scrollDirection = Axis.vertical,
+    ScrollPhysics? physics,
+    bool Function(OverscrollNotification)? callback,
```

Comment:
nit: use a typedef for this as per style guide?

Reply:
Actually, looks like this param is just unused? Remove it?

Reply:
Ah! I left that behind from an earlier test idea. :P Thanks!